internal/storage/runtime.go

```diff
@@ -71,6 +71,11 @@ type RuntimeServer interface {
 	// Pointer arguments can be nil.  Either the image name or ID can be
 	// omitted, but not both.  All other arguments are required.
 	CreatePodSandbox(systemContext *types.SystemContext, podName, podID, imageName, imageAuthFile, imageID, containerName, metadataName, uid, namespace string, attempt uint32, idMappingsOptions *storage.IDMappingOptions, labelOptions []string, privileged bool) (ContainerInfo, error)
+	// RemovePodSandbox deletes a pod sandbox's infrastructure container.
+	// The CRI expects that a sandbox can't be removed unless its only
+	// container is its infrastructure container, but we don't enforce that
+	// here, since we're just keeping track of it for higher level APIs.
+	RemovePodSandbox(idOrName string) error
```

Comment:
This function was removed by an earlier commit, and now it's being re-added. I don't follow...

Reply:
I have reworked the commits to only take the lint fixing parts of this commit. I believe this was readded by @saschagrunert to debug failures that other commits ended up fixing. we'll see how that works...